stingray/pulse/modeling.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+from astropy.modeling import models, fitting
+
+
+def sinc(x):
```

Comment:
I used numpy. However my definition was slightly different (a factor pi ), so I normalized the input variable.